.github/workflows/superset-helm-release.yml

```diff
@@ -0,0 +1,33 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
Let's only do this on master

Reply:
@craig-rueda Sure. This was only for the initial phase.
Do you by any chance want to release charts for any older versions of Superset?

Reply:
No, I think just going forward is fine